src/Alert/Alert.stories.js

```diff
@@ -1,32 +1,143 @@
 import React from 'react'
+import { withDesign } from 'storybook-addon-designs'
 import { Alert } from './Alert'
+import { Button } from '../Button/Button'
+import { alertTypes } from './Alert.levers'
+import { textMixed } from '../../.storybook/stories/Paragraph.stories'
+import DOMPurify from 'dompurify'
 
 export default {
   title: 'Composite/Alert',
-  component: Alert
+  decorators: [withDesign],
+  component: Alert,
+  subcomponents: { Button },
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/file/Kmd4utmJFPRMVeCFEEBQhLtx/Decanter-Design-System?node-id=8095%3A0'
+    },
+    docs: {
+      description: {
+        component: 'For displaying a notification that keeps people informed of a status, or for displaying a validation message that alerts someone of an important piece of information.'
+      }
+    }
+  },
+  argTypes: {
+    type: {
+      control: {
+        type: 'select',
+        options: alertTypes
+      }
+    }
+  }
```

Comment:
If you put these options on the default export they are considered 'global' for the component.